Compilation fails on latest experimental (cdda-experimental-2024-11-24-1805)

### Describe the bug

Compilation fails due to this error
`src/ranged.cpp:1245:8: error: no previous declaration for ‘double calculate_aim_cap_without_target(const Character&, const tripoint_bub_ms&)’ [-Werror=missing-declarations]
 1245 | double calculate_aim_cap_without_target( const Character &you, const tripoint_bub_ms &target )`

### Attach save file

N/A

### Steps to reproduce

1. make RELEASE=1 TILES=1 SOUND=1 LOCALIZE=0 BACKTRACE=0 USE_XDG_DIR=1 ASTYLE=0 TESTS=0
2. make: *** [Makefile:1107: obj/tiles/ranged.o] Error 1

### Expected behavior

errorless compilation

### Screenshots

_No response_

### Versions and configuration

OS - latest artix linux

### Additional context

build options: 
RELEASE=1 TILES=1 SOUND=1 LOCALIZE=0 BACKTRACE=0 USE_XDG_DIR=1 ASTYLE=0 TESTS=0
git reflog:
71b3dbe219 (HEAD -> master, tag: cdda-experimental-2024-11-24-1805, origin/master, origin/HEAD) HEAD@{0}: clone: from https://github.com/CleverRaven/Cataclysm-DDA.git
(this clone is blobless)

--- a/src/ranged.cpp
+++ b/src/ranged.cpp
@@ -1242,7 +1242,7 @@ int throw_cost( const Character &c, const item &to_throw )
     return std::max( 25, move_cost );
 }
 
-double calculate_aim_cap_without_target( const Character &you, const tripoint_bub_ms &target )
+static double calculate_aim_cap_without_target( const Character &you, const tripoint_bub_ms &target )
 {
     const int range = rl_dist( you.pos_bub(), target );
     // Get angle of triangle that spans the target square.

--- a/src/ranged.cpp
+++ b/src/ranged.cpp
@@ -1242,7 +1242,8 @@ int throw_cost( const Character &c, const item &to_throw )
     return std::max( 25, move_cost );
 }
 
-static double calculate_aim_cap_without_target( const Character &you, const tripoint_bub_ms &target )
+static double calculate_aim_cap_without_target( const Character &you,
+        const tripoint_bub_ms &target )
 {
     const int range = rl_dist( you.pos_bub(), target );
     // Get angle of triangle that spans the target square.

